src/styles/index.scss

```diff
@@ -1,3 +1,150 @@
-body {
+* {
```

Comment:
It's better not to use `*` selector for styles, because it might be not good for performance.